x/gamm/pool-models/balancer/pool.go

```diff
@@ -17,14 +17,15 @@ import (
 
 //nolint:deadcode
 const (
-	errMsgFormatSharesAmountNotPositive       = "shares amount must be positive, was %d"
-	errMsgFormatTokenAmountNotPositive        = "token amount must be positive, was %d"
-	errMsgFormatTokensLargerThanMax           = "%d resulted tokens is larger than the max amount of %d"
-	errMsgFormatSharesLargerThanMax           = "%d resulted shares is larger than the max amount of %d"
+	nonPostiveSharesAmountErrFormat = "shares amount must be positive, was %d"
+	nonPostiveTokenAmountErrFormat  = "token amount must be positive, was %d"
+	sharesLargerThanMaxErrFormat    = "%d resulted shares is larger than the max amount of %d"
+	invalidInputDenomsErrFormat     = "input denoms must already exist in the pool (%s)"
+
+	// the errors below are being used in export_test
```

Comment:
I think this comment is misleading?(called in `amm_test.go`)

Reply:
removed